# Generated by Django 5.0.1 on 2025-01-24 00:57

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('outbound', '0002_alter_outbounddelivery_options_and_more'),
    ]

    operations = [
        migrations.CreateModel(
            name='Place',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=200, unique=True)),
                ('latitude', models.DecimalField(decimal_places=6, help_text='Latitud de la ubicación.', max_digits=9)),
                ('longitude', models.DecimalField(decimal_places=6, help_text='Longitude de la ubicación.', max_digits=9)),
                ('city', models.CharField(blank=True, max_length=100, null=True)),
                ('contact_person', models.CharField(blank=True, max_length=100, null=True)),
                ('contact_phone', models.CharField(blank=True, max_length=15, null=True, validators=[django.core.validators.RegexValidator(message='Número de teléfono inválido.', regex='^\\+?1?\\d{9,15}$')])),
            ],
            options={
                'verbose_name': 'Lugar',
                'verbose_name_plural': 'Lugares',
                'db_table': 'outbound_place',
            },
        ),
    ]
